src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -1328,6 +1331,35 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
                     self.highlightedLegendItems.clear();
                 });
             },
+            get colorByStructuralVariant() {
+                // radio buttons in waterfall plot
+                if (self.plotType.result === PlotType.WaterfallPlot) {
+                    return (
+                        this._colorByStructuralVariant &&
+                        !this._colorByCopyNumber &&
+                        !this.colorByMutationType
+                    );
+                } else {
+                    return this._colorByStructuralVariant;
+                }
+            },
+            get _colorByStructuralVariant() {
```

Comment:
colorByStructuralVariant AND _colorByStructuralVariant?

Reply:
yes, so the `_colorBy` members represent the actual state from the URL, the `colorBy` members are what is read by the app which converts the "checkbox" logic into "radio button" logic for the waterfall plot where only one can be on at once.

Reply:
lets just put this bit of logic in the "public" one to avoid the confusing duplication

Reply:
rename this to ...fromUrl